project/dexter-vs/Dexter/Core/UI/Settings/SettingsControl.cs

```diff
@@ -198,9 +198,42 @@ DexterInfo IDexterInfoProvider.Load()
             return dexterInfo;
         }
 
+        private void createUser()
+        {
+            // We need to save current url 
+            DexterInfo dexterInfo = GetDexterInfoFromSettings();
+            var result = "";
+
+            if (!validator.ValidateServerConnection(dexterInfo, out result))
+            {
+                MessageBox.Show("Couldn't connect to Dexter server. Setting analysis mode to standalone", "Dexter warning", MessageBoxButtons.OK, MessageBoxIcon.Warning);
+                standaloneCheckBox.Checked = true;
+                dexterInfo.standalone = true;
+                Save(dexterInfo);
+            }
+            else if (validator.ValidateUserCredentials(dexterInfo, out result))
+            {
+                MessageBox.Show($"Couldn't create user: user {dexterInfo.userName} already exists", "Dexter warning", MessageBoxButtons.OK, MessageBoxIcon.Warning);
+            }
+            else
+            {
+                DexterClient.Instance = new DexterClient(new DexterHttpClientWrapper(dexterInfoProvider), dexterInfoProvider);
+                DexterClient.Instance.AddAccount(dexterInfo.userName, dexterInfo.userPassword, false);
+                MessageBox.Show(string.Format("Created new user {0}", dexterInfo.userName), "Dexter info", MessageBoxButtons.OK, MessageBoxIcon.Information);
```

Comment:
Because AddAccount() is async function, 
The message box is shown before the completion of this function.

The message box should be shown after checking the completion of AddAccount() fhunction.
And if the account creation fails, the message box might display a message related to this.